mod_test.ts

```diff
@@ -949,25 +949,47 @@ const TXT_PLUGIN: esbuildNative.Plugin = {
   },
 };
 
-Deno.test("txt plugin", async (t) => {
+Deno.test("css plugin", async (t) => {
   +await testLoader(t, LOADERS, async (esbuild, loader) => {
     const res = await esbuild.build({
       ...DEFAULT_OPTS,
       plugins: [
-        denoResolverPlugin(),
-        TXT_PLUGIN,
-        denoLoaderPlugin({ loader }),
+        ...denoPlugins({ loader }),
+        CSS_PLUGIN,
       ],
-      entryPoints: ["./testdata/hello.txt"],
+      entryPoints: ["./testdata/css.ts"],
+      bundle: true,
```

Comment:
I decided to add bundle: true since the entry point is typescript that is importing the css file.